GameServer.py

```diff
@@ -6,35 +6,44 @@
 from PodSixNet.Server import Server
 
 from ClientHandler import ClientHandler
-
+from PlayerInfo import PlayerInfo
 
 class GameServer(Server):
     channelClass = ClientHandler
+    idArray = [0, 0, 0]
+    playersDict = []
 
     def __init__(self, *args, **kwargs):
```

Comment:
Still not sure about `playersDict` name, and still not sure about having both WeakKeyDictionary and Dictionary for the same data.

In `AddPlayer()` we append to `playersDict` but in `DelPlayer()` we don't delete it. 

So we should probably either get rid of WeakKeyDictionary and stick with normal Dictionary, or find something better

Reply:
You are right, I changed "playersDict" to "playersInfoArray", which makes it clear.

I also didn't manage how to properly delete players, thats why we only add players for now.

I would stick with array of dictionaries as well as WeakKeyDictionary, because it's easy to send it to other clients. In array of dictionaries we hold playersInfo and in WeakKeyDictionary we keep clients channels. I added deleting not in not only WeakKeyDictionary but also in playersInfoArray